docs/extensions/netsuite/considerations.md

```diff
@@ -28,6 +28,14 @@ Request payload example:
 }
 ```
 
+### CURL version example
+```bash
+curl --location --request PATCH 'https://[EAMPLE].rossum.app/api/v1/hooks/[HOOK_ID]' \
+--header 'Authorization: token [API_TOKEN]' \
+--header 'Content-Type: application/json' \
+--data '{"token_lifetime_s": 120}'
```

Comment:
Shouldn't this be:

```suggestion
--data '{"timeout_s": 120}'
```

🤔

Reply:
You're right, the correct parameter for timeout is "timeout_s"